src/components/form/adapters/widget-adapter.ts

```diff
@@ -1,15 +1,14 @@
 import React from 'react';
 import { WidgetProps } from '../widgets/types';
-import { LimeElementsAdapter } from './base-adapter';
 import { capitalize } from 'lodash-es';
 import { getHelpComponent } from '../help';
 
-interface WidgetAdapterProps {
+export interface WidgetAdapterProps {
     name: string;
     value?: any;
     widgetProps: WidgetProps;
     extraProps?: any;
-    events?: { [key: string]: (event: any) => void };
+    events: { change: (event: CustomEvent) => void };
```

Comment:
Hmm… this changes the object from allowing any number of properties (`[key: string]`), to only allowing a single property called `event`. Was that really intentional?

How about the following?

```ts
    events: {
        change: (event: CustomEvent) => void,
        [key: string]: (event: any) => void,
    };
```

This makes the object required, and it has to have the `change` property, but it can also have any number of extra properties.

Although, come to think of it, this commit doesn't change `lime-elements.api.md`, so I guess this is a purely internal change anyway?

Reply:
> Was that really intentional?

Yes. That’s how it’s actually used, and if we need to handle more events, we can add them. It’s clearer what events need to be handled and easier to implement than allowing any event.

> so I guess this is a purely internal change anyway?

Correct